1-verify-prerelease

```diff
@@ -3,6 +3,13 @@
 # Compare the list of packages in pre-release for against the release
 # list put together by the OSG Release Coordinator
 
+
+
+. release-common.sh
```

Comment:
You already source release-common.sh a few lines down.

Reply:
Removed